checkov/common/checks_infra/solvers/attribute_solvers/equals_attribute_solver.py

```diff
@@ -7,9 +7,9 @@
 class EqualsAttributeSolver(BaseAttributeSolver):
     operator = Operators.EQUALS  # noqa: CCE003  # a static attribute
 
-    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
-        super().__init__(resource_types=resource_types,
-                         attribute=attribute, value=value)
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any, is_jsonpath_check: bool = False) -> None:
```

Comment:
like @nimrodkor wrote, you can remove almost everywhere the `__init__` method, because it doesn't do anything and the parent class one will be invoked anyway.